plugins/plugin-core-support/src/lib/tab-completion.ts

```diff
@@ -577,10 +602,14 @@ const suggest = (param, last: string, block: HTMLElement, prompt: HTMLInputEleme
   if (param.file) {
     // then the expected parameter is a file; we can auto-complete
     // based on the contents of the local filesystem
+    debug('lsssss', path.basename(last))
```

Comment:
what's lsssss?

Reply:
oops I forgot to clean up the debug info